clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -3984,6 +3984,10 @@ def warn_thread_attribute_decl_not_lockable : Warning<
 def warn_thread_attribute_decl_not_pointer : Warning<
   "%0 only applies to pointer types; type here is %1">,
   InGroup<ThreadSafetyAttributes>, DefaultIgnore;
+def warn_thread_attribute_not_on_scoped_lockable_param : Warning<
+  "%0 attribute applies to function parameters only if their type is a "
+  "reference to a 'scoped_lockable'-annotated type">,
+  InGroup<ThreadSafetyAttributes>, DefaultIgnore;
```

Comment:
Why is this diagnostic off by default? It seems like this is a case users wouldn't want the attribute to be silently ignored.

Reply:
The `DefaultIgnore` setting aligns with the fact that the warnings `-Wthread-safety-analysis` and `-Wthread-safety-attributes` are not enabled by default.